README.md

```diff
@@ -1,2 +1,188 @@
-# rails-budget-app
- 
+<a name="readme-top" align="center"></a>
+
+<div align="center">
+  <h1>Money Track App</h1>
+  <br/>
+
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  <!-- - [Deployment](#triangular_flag_on_post-deployment) --> 
+- [👥 Authors](#authors)
+<!-- - [🔭 Future Features](#future-features) -->
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 Money Track <a name="about-project"></a>
+Money track is a mobile web application where you can manage your budget: you have a list of transactions associated with a category, so that you can see how much money you spent and on what.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Server</summary>
+  <ul>
+    <li><a href="https://www.ruby-lang.org/en/">Ruby</a></li>
+  </ul>
+</details>
+
+<details>
+  <summary>Database</summary>
+  <ul>
+    <li><a href="https://www.postgresql.org/">PostgreSQL</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+- This Ruby on Rails application allows the user to:
+   - register and log in, so that the data is private to them.
+   - introduce new transactions associated with a category.
+   - see the money spent on each category.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+## 🚀 Live Demo <a name="live-demo"></a>
```

Comment:
- [ ] Kindly redeploy your project and make sure it's from the 'dev' branch. Also, make sure postgres is rightly configured when deploying. 
- You can use this link: https://medium.com/@laura_brooks/how-to-deploy-a-ruby-on-rails-app-with-render-com-9a824193e636
- The above has a guide.
- [ ] Your your video, kindly use another medium like screen recorder if you're not getting acquainted with loom. These skills are needed to move forward 👍💯

Reply:
followed the same process for render but was not working